### PR TITLE
Update script for composer.

### PR DESCRIPTION
**Pull request by sylus**
Largely based on the work @ https://github.com/docker-solr/docker-solr/blob/master/update.sh

### CODE REVIEW

**Comment by sylus** [comment]
Tested so far by deleting the following directories then re-generating:

```sh
rm -rf 1.0.1
./update 1.0.1
git diff
```

```sh
rm -rf 1.0.0-beta2
./update 1.0.0-beta2
git diff
```

File: update.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@
 #
 # Usage: ./update.sh x.y.z
 #
-# This script runs to create a Dockerfile for a new Composer version.
-# If you specify a partial version, like '1' or '1.0', it will determine the most recent sub version like 1.0.1.
-set -e
+# This script runs to create a Dockerfile for a new Drush version.
```
Comment (RobLoach) [inline update.sh]: Composer versions :wink:

**Comment by RobLoach** [comment]
Looks good! Anything else that we need? Thinking it's merge time!  :+1:

**Comment by sylus** [comment]
I ran a few tests and seems to be working.

a) Scenario 1 we specify 7.0 so it will look for latest of that branch so finds 7.0.0
b) Scenario 2 we specify 7 so it will look for latest of that branch so find 7.3.0
c) Scenario 3 we specify 7.0.0-rc2 it will look for exactly that branch and use it

## Scenario 1 

```sh
$ ./update 1.1.0
```

```patch
diff --git a/1.1.0/Dockerfile b/1.1.0/Dockerfile
new file mode 100644
index 0000000..c906da9
--- /dev/null
+++ b/1.1.0/Dockerfile
@@ -0,0 +1,11 @@
+# Composer Docker Container
+FROM composer/composer:base
+MAINTAINER Rob Loach <robloach@gmail.com>
+
+ENV COMPOSER_VERSION 1.1.0
+
+# Install Composer
+RUN php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && rm -rf /tmp/composer-setup.php
+
+# Display version information.
+RUN composer --version
diff --git a/1.1.0/alpine/Dockerfile b/1.1.0/alpine/Dockerfile
new file mode 100644
index 0000000..02d8d6d
--- /dev/null
+++ b/1.1.0/alpine/Dockerfile
@@ -0,0 +1,11 @@
+# Composer Docker Container
+FROM composer/composer:base-alpine
+MAINTAINER Rob Loach <robloach@gmail.com>
+
+ENV COMPOSER_VERSION 1.1.0
+
+# Install Composer
+RUN php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && rm -rf /tmp/composer-setup.php
+
+# Display version information.
+RUN composer --version
```

## Scenario 2 

```sh
$ ./update 1.0.1
```

```patch
no changes
```


## Scenario 3 

```sh
$ ./update 1.1.0-RC
```

```patch
diff --git a/1.1.0-RC/Dockerfile b/1.1.0-RC/Dockerfile
new file mode 100644
index 0000000..aa5c8a3
--- /dev/null
+++ b/1.1.0-RC/Dockerfile
@@ -0,0 +1,11 @@
+# Composer Docker Container
+FROM composer/composer:base
+MAINTAINER Rob Loach <robloach@gmail.com>
+
+ENV COMPOSER_VERSION 1.1.0-RC
+
+# Install Composer
+RUN php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && rm -rf /tmp/composer-setup.php
+
+# Display version information.
+RUN composer --version
diff --git a/1.1.0-RC/alpine/Dockerfile b/1.1.0-RC/alpine/Dockerfile
new file mode 100644
index 0000000..34d8969
--- /dev/null
+++ b/1.1.0-RC/alpine/Dockerfile
@@ -0,0 +1,11 @@
+# Composer Docker Container
+FROM composer/composer:base-alpine
+MAINTAINER Rob Loach <robloach@gmail.com>
+
+ENV COMPOSER_VERSION 1.1.0-RC
+
+# Install Composer
+RUN php /tmp/composer-setup.php --no-ansi --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} && rm -rf /tmp/composer-setup.php
+
+# Display version inform
```

**Comment by RobLoach** [comment]
Nicely done, @sylus! I've added you as a contributor on the project, feel free to issue PRs and such, or merge PRs as needed. :+1:

**Comment by sylus** [comment]
That is awesome! Love to help however I can :) Thx so much!

**Pull request merged**